include/deal.II/base/vectorization.h

```diff
@@ -1592,6 +1698,68 @@ vectorized_load_and_transpose(const unsigned int          n_entries,
 
 
 
+/**
+ * Specialization for float and AVX-512.
+ */
+template <>
+inline DEAL_II_ALWAYS_INLINE void
+vectorized_load_and_transpose(const unsigned int             n_entries,
+                              const std::array<float *, 16> &in,
+                              VectorizedArray<float, 16> *   out)
+{
+  // Similar to the double case, we perform the work on smaller entities. In
+  // this case, we start from 128 bit arrays and insert them into a full 512
+  // bit index. This reduces the code size and register pressure because we do
+  // shuffles on 4 numbers rather than 16.
+  const unsigned int n_chunks = n_entries / 4;
+
+  // To avoid warnings about uninitialized variables, need to initialize one
+  // variable to a pre-exisiting value in out, which will never get used in
+  // the end. Keep the initialization outside the loop because of a bug in
+  // gcc-9 which generates a "vmovapd" instruction instead of "vmovupd" in
```

Comment:
And given that I read this one here, maybe limit the statement to "gcc-9.1", because this was fixed in gcc very quickly after my bug report and is part of gcc-9.2 that is much more common nowadays.